src/pages/benefits/viewAllBenefit/ApplicationDetails.tsx

```
@@ -120,6 +122,57 @@ const ApplicationDetails: React.FC = () => {
+  const checkDocumentVerificationStatus = (docs: Document[]) => {
+    if (docs.length === 0) {
+      setIsDocumentsVerified(true); // No documents to verify
+      return true;
+    }
+    const allVerified = docs.every((doc) => doc.status === "Verified");
+    setIsDocumentsVerified(allVerified);
+    return allVerified;
+  };
+
+  // Helper function to check if eligibility has been evaluated
+  const checkEligibilityStatus = (criteria: any[]) => {
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)